mmv1/products/logging/OrganizationSettings.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'OrganizationSettings'
+description: |
+  Default resource settings control whether CMEK is required for new log buckets. These settings also determine the storage location for the _Default and _Required log buckets, and whether the _Default sink is enabled or disabled.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Configure default settings for organizations and folders': 'https://cloud.google.com/logging/docs/default-settings'
+  api: 'https://cloud.google.com/logging/docs/reference/v2/rest/v2/TopLevel/getSettings'
+base_url: 'organizations/{{organization}}/settings'
+self_link: 'organizations/{{organization}}/settings'
+import_format: ['organizations/{{organization}}/settings']
+# Hardcode the updateMask since d.HasChanged does not work on create.
+create_url: 'organizations/{{organization}}/settings?updateMask=disableDefaultSink,storageLocation,kmsKeyName'
+update_url: 'organizations/{{organization}}/settings?updateMask=disableDefaultSink,storageLocation,kmsKeyName'
+# This is a singleton resource that already is created, so create
+# is really an update, and therefore should be PATCHed.
+create_verb: :PATCH
+update_verb: :PATCH
+# This is a singleton resource that cannot be deleted, so skip delete.
+skip_delete: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "logging_organization_settings_all"
+    primary_resource_id: "example"
+    skip_test: true
```

Comment:
Could you add a comment above the `skip_test` line so we know why the test is skipped? Something like `# Covered by update test`